ESPLoader.js

```diff
@@ -280,7 +280,10 @@ class ESPLoader {
         var resp;
         this.chip = null;
         this.write_char('Connecting...');
-        await this.transport.connect();
+        await this.transport.connect().catch((e) => {
```

Comment:
Can we check the state before trying to call connect?

Reply:
I think it's better to update `webserial.js` to store if the connection is openened

Reply:
I'll add it to `webserial.js` instead.